config.json

```diff
@@ -20,6 +23,12 @@
             "Class" : "DNWS.DelayPlugin",
             "Preprocessing" : "false",
             "Postprocessing" :"false" 
+        },
+		{
+            "Path" : "clientinfo",
+            "Class" : "DNWS.ClientInfoPlugin",
```

Comment:
Did you really test the code?  Your class name is "public class InfoPlugin : IPlugin", but your register the plugin as DNWS.ClientInfoPlugin.   I don't think it is working here.